llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -735,6 +739,9 @@ PassBuilder::buildModuleSimplificationPipeline(OptimizationLevel Level,
                                                ThinOrFullLTOPhase Phase) {
   ModulePassManager MPM;
 
+  if (SYCLOptimizationMode)
```

Comment:
I'd suggest to add this pass to clang\lib\CodeGen\BackendUtil.cpp, similar to other SYCL-specific passes like `createSYCLLowerWGLocalMemoryLegacyPass`

Reply:
Done